.github/workflows/R-CMD-check.yaml

```
@@ -1,22 +1,21 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+#
+# NOTE: This workflow is overkill for most R packages and
+# check-standard.yaml is likely a better choice.
+# usethis::use_github_action("check-standard") will install it.
 on:
   push:
-    branches:
-      - main
-      - master
-      - dev
+    branches: [main, master]
   pull_request:
-    branches:
-      - main
-      - master
-      - dev
-  schedule:
-    - cron:  '0 8 * * *'
+    branches: [main, master]
 
-name: R-CMD-check
+name: R-CMD-check.yaml
```

```suggestion
name: R-CMD-check
```

I'd really like to drop the .yaml from the name. @aylapear what do you think?

---

Yeah, that is a great catch Joe. We should file an issue with them if it hasn't already been done.

---

OK - great -I've filed an issue in our internal infrastructure repository for this.